src/components/TextInput/TextInput.js

```diff
@@ -258,10 +276,15 @@ class TextInput extends React.Component<Props, State> {
               paddingRight: (inputProps.multiline ? 58 : 14) + additionalRightPadding,
               paddingTop: inputProps.multiline ? 10 : 0,
               textAlignVertical: inputProps.multiline ? 'top' : 'center',
-              marginBottom: 10,
             }}
             onLayout={onLayout}
           />
+          <RNInput
```

Comment:
I'm a little bit confused now - will it render two input fields now (one below another)?

Reply:
This hack focuses on a transparent input first which is a single line input and on ios it brings the keyboard avoiding scrollview up above the device keyboard. After the animation timing it focuses back on the multiline input.